### PR TITLE
Search icon

### PR DESCRIPTION
**Pull request by jennalandy**
Adding a dark option for the search icon to use in our user interface for keyboard shortcut settings.

### CODE REVIEW

**Comment by afshin** [comment]
Hi @jennalandy! Could you please remove the `package-lock.json` file and also add a screenshot? We're not currently using package lock files.

**Comment by jennalandy** [comment]
Hi @afshin I just removed package-lock.json. What do you want a screenshot of?

**Comment by jasongrout** [comment]
> What do you want a screenshot of?

Something that shows the visual change (before/after preferable). When we have a UI change, we try to post screenshots in the PR to (a) document the change, and (b) make it easier for the reviewer to see the intended effect.

**Comment by jennalandy** [comment]
Hi @jasongrout @afshin, I haven't changed any existing UI, just adding the icon for use in the extension my team is working on! Here's a screenshot of the darker search icon (it's the same one that's used in the command palette search bar with the color used for the close icons).

search

**Comment by ellisonbg** [comment]
Can we merge this for 0.33? @jasongrout

File: packages/theme-dark-extension/style/urls.css [hunk 1]
```diff
@@ -62,7 +62,8 @@
   --jp-icon-inverse-circle: url('icons/md/circle-inverse.svg');
   --jp-icon-close-circle: url('icons/md/close-circle.svg');
   --jp-icon-inverse-close-circle: url('icons/md/close-circle-inverse.svg');
-  --jp-icon-search: url('icons/md/search.svg');
+  --jp-icon-search: url('icons/md/search-white.svg');
+  --jp-icon-search-white: url('icons/md/search.svg');
```
Comment (ian-r-rose) [inline packages/theme-dark-extension/style/urls.css]: This is confusing to me. Why is `--jp-icon-search` pointing to `search-white.svg` and why is `--jp-icon-search-white` pointing to `search.svg`?

Comment (ian-r-rose) [reply packages/theme-dark-extension/style/urls.css]: Perhaps it should be named `--jp-icon-inverse-search`, etc?

Comment (jennalandy) [reply packages/theme-dark-extension/style/urls.css]: @ian-r-rose They're swapped in dark theme for when the background color inverts. I can definitely change it to --jp-icon-inverse-search if you think that would be more intuitive!

**Comment by ian-r-rose** [comment]
Hi @jennalandy, I took a closer look, and have a better grasp on why you are specifying it as `white` rather than `inverse`, so consider that suggestion withdrawn. There was a redundant icon in the `theme-dark-extension` which was the source of some of my confusion, so I went ahead and removed that so we can get this in for the release today.

**Comment by jennalandy** [comment]
Alright thank you!

**Comment by ian-r-rose** [comment]
Does that change make sense for what your are trying to do?

**Comment by jennalandy** [comment]
Yeah, I was getting a warning in the apputils test for having two pointers to the same icon which is why I added the redundant one. This is passing all tests though so I guess it wasn't necessary. It definitely makes more sense this way!

**Comment by ian-r-rose** [comment]
Okay, I think we should merge this, anything else we need for your work to move forwards?

**Comment by jennalandy** [comment]
We have one small change we'd like to make to the keyboard shortcut extension schema, I'll create a pull request right now but don't worry if there's not enough time for it

**Comment by ian-r-rose** [comment]
Thanks!

**Pull request merged**